### DISCUSSION TITLE
Results include items with score greater than threshold

### DISCUSSION THREAD

**Discussion by bschlenk** [krisk/Fuse#621] [Q&A] [answered]
I'm kinda confused about what threshold is supposed to do. The results always include items with scores higher than the threshold. I thought it was supposed to limit to results with a score <= the threshold. 

Try pasting this into `main.js` on the [live demo page](https://fusejs.io/demo.html):

```js
const options = {
  // isCaseSensitive: false,
  includeScore: true,
  // shouldSort: true,
  // includeMatches: false,
  // findAllMatches: false,
  // minMatchCharLength: 1,
  // location: 0,
  threshold: 0.3,
  // distance: 100,
  // useExtendedSearch: false,
  // ignoreLocation: false,
  // ignoreFieldNorm: false,
  // fieldNormWeight: 1,
  keys: [
    "title",
    "author.firstName"
  ]
};

const fuse = new Fuse(list, options);

// Change the pattern
const pattern = "s"

return fuse.search(pattern)
```

The results include scores up to .47

**Comment by krisk** [comment, marked as answer]
Take a look at the [scoring theory](https://fusejs.io/concepts/scoring-theory.html) to understand how the score is calculated 😄
